.github/workflows/clang_format.yml

```diff
@@ -27,10 +27,31 @@ jobs:
           # make sure the parent commit is grabbed as well, because
           # that's what will get formatted (i.e. the most recent commit)
           fetch-depth: 2
+      # retrieve comma-separated list of modified files
+      - name: Retrieve changed files
+        id: changed-files
+        uses: tj-actions/changed-files@v42.0.5
+        with:
+          files: |
```

Comment:
This glob needs to be improved, we also have cpp source files directly under `src`. Also under `include`

Reply:
@FlayaN you think you can handle the globs easily?

Reply:
Hello! Like I said I will be going on a 2 week vacation later today, so I don't have time to fix the globs (And test them today). I can however guess :P 

Looking at documentation at: https://github.com/tj-actions/changed-files

It looks like they support the `src/**` glob syntax.
So my guess for a good glob would be:
```yaml
files: |
  src/**
  include/**
  **/*.hlsl
  **/*.hlsli
```

Reply:
@FlayaN since you're on a roll, you want to take this over the line?

Reply:
I can give this another look in a couple days

Reply:
I recently updated the glob with the help of https://codepen.io/mrmlnc/pen/OXQjMe:
```
  files: |
    src/**/*.{cpp,h}
    include/**/*.{h,hpp}
    **/*.hlsl
    **/*.hlsli
```

Here's a couple example runs of the updated glob on [rjwignar/issue-128-test](https://github.com/rjwignar/skyrim-community-shaders/blob/4671df22cad14a3b61375858c0f0ef63d6f2f998/.github/workflows/clang_format.yml#L35-L40):
| Test | Commit | Clang-Format Job |
| --- | --- | --- |
| Update cpp/h files in `src` | https://github.com/rjwignar/skyrim-community-shaders/commit/c17a2f6b6b06ef5af99d2789ba1d0b25806a3c5e | https://github.com/rjwignar/skyrim-community-shaders/actions/runs/9409659767/job/25920004938 |
| Update h/hpp files in `include` | https://github.com/rjwignar/skyrim-community-shaders/commit/51b932281a38d957481e9e7b2dab1912644faa1c | https://github.com/rjwignar/skyrim-community-shaders/actions/runs/9409737196/job/25920220892 |

The recent force-push was due to a rebase of my branch (`rjwignar/issue-128`) on top of the latest `dev` branch
Please let me know if additional changes are required.